src/Service/Bulk/BulkUserCreationService.php

```diff
@@ -281,23 +260,27 @@ private function getAllLineItemSlugs(): void
         if (empty($lineItems)) {
             throw new LineItemNotFoundException('No line items were found in database.');
         }
+
         $this->generateSlugData($lineItems);
     }
 
     private function getLastUserAssignedToLineItems(
         array $lineItemSlugs
     ): array {
         $userNameIncrementArray = [];
+
         foreach ($lineItemSlugs as $slugKey => $slug) {
             $assignment = $this->assignmentRepository->findByLineItemId($slugKey);
             $userNameIncrementArray[$slug] = self::DEFAULT_USERNAME_INCREMENT_VALUE;
+
             if (!empty($assignment)) {
-                $userInfo = $assignment->getUser()->getUsername();
-                $userNameArray = explode('_', (string)$userInfo);
-                $userNameLastNo = preg_match('/^\d+$/', end($userNameArray))
+                $userData = $assignment->getUser()->getUsername();
+                $userNameArray = explode('_', (string)$userData);
```

Comment:
I think it can be just `$userNameParts = explode('_', (string) $assignment->getUser()->getUsername());`

Reply:
Also please make sure every cast like (string) have a space after it.